zer0-os/zOS
# feat: add rewards feature flag

### What does this do?
- Hides the entire Rewards feature behind a feature flag (including notifications, etc).

### Why are we making this change?
- In light of recent information that some users have located the URL for production ZOS, we'd like to hide Rewards behind a feature flag so that they don't mistakenly think they are earning $ZERO that they are not actually earning.

### How do I test this?
- open the app and ensure that the rewards button is not visible > to view the rewards open the dev console and enter `window.FEATURE_FLAGS.resetPasswordPage = true` > refresh page. Rewards feature should now be visible. 

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?

https://github.com/zer0-os/zOS/assets/39112648/e3cabf9d-af7e-41d8-947d-2f3a4b8684b4
